ssl/packet_locl.h

```diff
@@ -102,6 +102,23 @@ __owur static ossl_inline int PACKET_equal(const PACKET *pkt, const void *ptr,
 }
 
 /*
+ * Similar to PACKET_equal() except that it compares only a limited number
+ * of bytes (i.e. as strncmp() is to strcmp()). And that it consumes the
+ * compared bytes on success.
+ */
+__owur static ossl_inline int PACKET_nequal(PACKET *pkt, const void *ptr,
```

Comment:
Not sure about the name. Perhaps `PACKET_equaln` might be better - I automatically read "nequal" as "not equal".